packages/jupyter/package.json

```diff
@@ -28,14 +27,16 @@
     "buffer": "^6.0.3",
     "classnames": "^2.3.2",
     "myst-common": "^0.0.16",
-    "myst-config": "^0.0.12",
+    "myst-config": "^0.0.14",
+    "myst-frontmatter": "^0.0.13",
     "myst-spec": "^0.0.4",
     "nanoid": "^4.0.0",
     "nbtx": "^0.2.3",
     "react-popper": "^2.3.0",
     "react-syntax-highlighter": "^15.5.0",
     "swr": "^1.3.0",
-    "thebe-core": "^0.1.1",
+    "thebe-core": "^0.1.6",
+    "thebe-react": "^0.0.7",
```

Comment:
@stevejpurves I think it is worth pinning the thebe packages to the same release number?! This is how we are doing it in theme, and is a few less numbers that we have to keep in mind. :)